src/__tests__/DataHandler.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
- [ ] It would be better if we write unit test for pure functions in our Redux code